### ISSUE TITLE
CI documentation problems

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#108]
- [ ] not documented how restart works
- [x] Failed to run 2023-03-14-153713_enable_postgis with: permission denied to create extension "postgis", see https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-94/5/pipeline/
- [X] build jobs fail with:
```
+ sudo /usr/local/bin/permaplant-reset-ci.sh
+ sudo -u postgres psql -c 'DROP DATABASE cidb;'
ERROR:  database "cidb" is being accessed by other users
DETAIL:  There are 60 other sessions using the database.
script returned exit code 1
```

**Comment by lukashartl** [comment]
I've manually stopped the permpaplant-pr service on dev now and restarted some of the failed pipelines.

After merging #78 the service will be automatically stopped.
This was only a problem now, because there are two different version of the pipeline deployed. Merging #78 will take care of this specific problem.

Restart works just by restarting, there is nothing specific to do.

**Comment by markus2330** [comment]
I merged #78 to fix this problem but the documentation is still not satisfactory. It should be described how someone can restart the pipeline (where in GUI and passphrase) and how to report bugs (I would suggest to simply create issues and assign to you).

I don't really understand what is meant with the "Restrictions" and how does it apply for CI users. Please keep in mind that readers of [doc/ci.md](https://github.com/ElektraInitiative/PermaplanT/pull/78/files#diff-daa4fda467ebc1a3ac6c10516a4df5ed17f4421e75f3504b1206b1b6df5801f9) know nothing of the infrastructure, Jenkins or Jenkinsfile (the Jenkinsfile should be mentioned in the documentation, too).

**Comment by markus2330** [comment]
And there are not enough permissions on the database, I added

- [ ] Failed to run 2023-03-14-153713_enable_postgis with: permission denied to create extension "postgis", see https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-94/5/pipeline/

in the top post.

**Comment by lukashartl** [comment]
Master Deployment works now.

Had to do a

```
REASSIGN OWNED BY seedling TO garden;
```

because I changed the database user at one point during setup.

**Comment by lukashartl** [comment]
@ElektraInitiative/permaplant 
Sorry for the inconvenience, please rebase your PRs 🙏

**Comment by markus2330** [comment]
Thank you, it works now! :heart: 

#94 was the only one affected of above problem and passes now.

@lukashartl Which PRs do still need a rebase?

Looking forward to see documentation, I think this will also reduce many confusions once people know when and how to restart/rebase (rebase is needed in the case that Jenkinsfile modifications are needed to pass builds).

**Comment by markus2330** [comment]
Fixed in #126
